### PR TITLE
Add organization model

### PR DESCRIPTION
**Pull request by marstrong**

### CODE REVIEW

File: models/Organization.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema;
+
+const OrganizationSchema = new Schema({
+    handle: {
+        type: String,
+        required: true
+    },
+    createdAt: {
```
Comment (haydenlinder) [inline models/Organization.js]: add name

File: validation/organization.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const Validator = require('validator');
+const validText = require('./valid_text');
+const Organization = require('../models/Organization');
+
+
+module.exports = function validateNewOrg(data) {
+    let errors = {};
+
+    data.handle = validText(data.first_name) ? data.first_name : '';
+
```
Comment (haydenlinder) [inline validation/organization.js]: fix copy pasta mess

File: validation/organization.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    if (Validator.isEmpty(data.handle)) {
+        errors.handle = 'a new organzation must have a valid handle';
+    }
+
```
Comment (haydenlinder) [inline validation/organization.js]: change valid handle to handle

File: validation/organization.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    Organization.findOne({ handle: data.handle }).then(org => {
+        if (org) {
+            errors.organization = "organization already exists";
+            return res.status(400).json(errors);
```
Comment (haydenlinder) [inline validation/organization.js]: capitalize error messages

File: models/Organization.js (hunk 1 above)
Comment (marstrong) [reply models/Organization.js]: done.

File: validation/organization.js (hunk 2 above)
Comment (marstrong) [reply validation/organization.js]: done.

File: validation/organization.js (hunk 3 above)
Comment (marstrong) [reply validation/organization.js]: all error messages have been updated.

File: validation/organization.js (hunk 4 above)
Comment (marstrong) [reply validation/organization.js]: done.

**Pull request merged**